pontoon/contributors/views.py

```diff
@@ -187,6 +187,36 @@ def toggle_user_profile_attribute(request, username):
     return JsonResponse({"status": True})
 
 
+@login_required(redirect_field_name="", login_url="/403")
+@require_POST
+@transaction.atomic
+def toggle_theme(request, username):
+    user = get_object_or_404(User, username=username)
+    if user != request.user:
+        return JsonResponse(
+            {
+                "status": False,
+                "message": "Forbidden: You don't have permission to edit this user",
+            },
+            status=403,
+        )
+
+    theme = request.POST.get("theme", None)
+    allowed_themes = ["light", "dark", "system"]
+
+    if theme not in allowed_themes:
+        return JsonResponse(
+            {"status": False, "message": "Bad Request: Invalid theme"},
+            status=400,
+        )
```

Comment:
We should only have themes defined in one places. [Looks like](https://stackoverflow.com/questions/60351804/no-validation-on-field-choices-django-postgres) calling `full_clean()` before `save()` will do the validation "for free"?